learning_resources/serializers.py

```diff
@@ -34,9 +35,16 @@ class LearningResourceTopicSerializer(serializers.ModelSerializer):
     Serializer for LearningResourceTopic model
     """
 
+    channel_url = serializers.SerializerMethodField(read_only=True)
```

Comment:
drf_spectacular is not picking up that channel_url can be null. That is why you needed the  @ts-expect-error in ManageListDialogs.tsx
 
```suggestion
    channel_url = serializers.SerializerMethodField(read_only=True, allow_null=True)
```